diff --git a/node_modules/@maplibre/maplibre-react-native/maplibre-react-native.podspec b/node_modules/@maplibre/maplibre-react-native/maplibre-react-native.podspec
index 4eae50f..a7bc1d2 100644
--- a/node_modules/@maplibre/maplibre-react-native/maplibre-react-native.podspec
+++ b/node_modules/@maplibre/maplibre-react-native/maplibre-react-native.podspec
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 # Global Variable Defaults
-$MLRN_NATIVE_VERSION ||= "6.17.1"
+$MLRN_NATIVE_VERSION ||= "6.18.0"
 $MLRN_SPM_SPEC ||= {
   url: "https://github.com/maplibre/maplibre-gl-native-distribution",
   requirement: {
